src/test/steps/check-your-answers.ts

```diff
@@ -24,3 +24,25 @@ Given("I've completed enough questions correctly to get to the check your answer
     ceremonyPlace: 'Maputo',
   });
 });
+
+Given("I've completed enough questions correctly to get to the check your answers page as a joint applicant", () => {
```

Comment:
Should this be `async`?

```suggestion
Given("I've completed enough questions correctly to get to the check your answers page as a joint applicant", async () => {
```